examples/fields/SmartDateField.README.md

```diff
@@ -0,0 +1,61 @@
+# SmartDateField
+
+## Why
+
+The `SmartDateField` lets you worry about operations on `Date`s as objects, without worrying about parsing dates, or formatting dates.  Currently the `SmartDateField` can parse 9 date types without any format strings.  This lets you concentrate on business logic, without worrying about incoming date formats.  This let's you confidently perform arithmetic and comparisons in `compute`, `recordCompute`, `batchRecordsCompute`, and `validate` on `Date`s.  `SmartDateField` has an `fString` argument that controls how the date is serialized after `validate` is finished.  Flatfile users no longer have to convert in and out of dates to maintain formatting.
+
+## What
+
+There are comprehensive tests for `SmartDateField`, the most demonstrative being [DateSheet.spec.ts](./DateSheet.spec.ts) .  This shows how dates can be compared as first class objects in a `recordCompute` function, and then formatted to a different persistence formate.
```

Comment:
Missed the importance of this on the first pass!  Found the `fString` portion of the code pulled the concepts together.  Should that be mentioned here (see it is mentioned in the "Why" section or will someone picking this up understand how they can use `fString`?

Reply:
Great catch, changed the wording